nameko/testing/pytest.py

```diff
@@ -115,9 +120,18 @@ def create():
         )
         return vhost
 
+    @retry(for_exceptions=(HTTPError, socket.timeout), delay=1, max_attempts=9)
```

Comment:
Is there a significance to `max_attempts=9`? (seems rather a lot)

Reply:
not really -other than to give it around 10 secs :)